x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/endpoint_operations_analyst.ts

```diff
@@ -6,32 +6,75 @@
  */
 
 import type { Role } from '@kbn/security-plugin/common';
-import { getNoResponseActionsRole } from './without_response_actions_role';
 
 export const getEndpointOperationsAnalyst: () => Omit<Role, 'name'> = () => {
-  const noResponseActionsRole = getNoResponseActionsRole();
+  // IMPORTANT
+  // This role is sync'ed with the role used for serverless and should not be changed
+  // unless the role for serverless has also been changed.
+  // This role is the default login for cypress tests as well (defend workloads team)
   return {
-    ...noResponseActionsRole,
+    elasticsearch: {
+      cluster: [],
+      indices: [
+        {
+          names: [
+            'metrics-endpoint.metadata_current_*',
+            '.fleet-agents*',
+            '.fleet-actions*',
+            'apm-*-transaction*',
+            'traces-apm*',
+            'auditbeat-*',
+            'endgame-*',
+            'filebeat-*',
+            'logs-*',
+            'packetbeat-*',
+            'winlogbeat-*',
+            '.lists*',
+            '.items*',
```

Comment:
I believe these should be `.lists-*` and `.items-*`

Reply:
I'm going to leave it as is, since it matches whats [defined in Project Controller](https://github.com/elastic/project-controller/blob/69ed9829dc53bedac0f92781070dcc64073c79b4/internal/project/security/config/roles.yml#L486-L487C1) (cc/ @dhurley14 )